src/app/components/OptimizelyPageMetrics/index.test.jsx

```diff
@@ -86,6 +86,68 @@ afterEach(() => {
   observers.clear();
 });
 
+describe('Optimizely Page View tracking', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should call Optimizely track function for Article Page on page render', async () => {
+    useOptimizelyVariation.mockReturnValue('variation_1');
+
+    render(
+      <ContextWrap pageType={ARTICLE_PAGE} service="news" isAmp={false}>
+        <OptimizelyPageMetrics pageView />
+      </ContextWrap>,
+    );
+
+    await waitFor(() => {
+      expect(optimizely.track).toHaveBeenCalledTimes(1);
```

Comment:
Do we need to include any additional parameters here to ensure that it's definitely a page view track which has been invoked?

```suggestion
      expect(optimizely.track).toHaveBeenCalledTimes(1);
      expect(optimizely.track).toHaveBeenCalledWith('page-views');
```

Reply:
I've gone down a different route with OptimizelyPageMetrics/index.test.jsx. Hopefully the existing tests in the nested components cover the Optimizely logic (e.g. calling Opimizely.track).

Instead, OptimizelyPageMetrics/index.test.jsx now just checks the isAmp logic and that each nested component is called correctly. 

I hope this ok. I used AI to help. It's certainly easier to read.

Reply:
I'm fine with that - whatever is easiest & simplest.